DscResource.DocumentationHelper.psd1

```diff
@@ -0,0 +1,119 @@
+#
+# Module manifest for module 'DscResource.DocumentationHelper'
+#
+# Generated by: Brian Farnhill
+#
+# Generated on: 02/08/2016
+#
+
+@{
+
+    # Script module or binary module file associated with this manifest.
+    # RootModule = ''
+
+    # Version number of this module.
+    ModuleVersion = '1.0.0.0'
+
+    # ID used to uniquely identify this module
+    GUID = 'bc93d075-382c-4f3b-8e06-1e0492394728'
+
+    # Author of this module
+    Author = 'Microsoft Corporation'
+
+    # Company or vendor of this module
+    CompanyName = 'Microsoft Corporation'
+
+    # Copyright statement for this module
+    Copyright = '(c) 2015-2016 Microsoft Corporation. All rights reserved.'
+
+    # Description of the functionality provided by this module
+    Description = 'This module is used to assist in generating documentation for PowerShell DSC resoruces'
+
+    # Minimum version of the Windows PowerShell engine required by this module
+    # PowerShellVersion = ''
+
+    # Name of the Windows PowerShell host required by this module
+    # PowerShellHostName = ''
+
+    # Minimum version of the Windows PowerShell host required by this module
+    # PowerShellHostVersion = ''
+
+    # Minimum version of Microsoft .NET Framework required by this module
+    # DotNetFrameworkVersion = ''
+
+    # Minimum version of the common language runtime (CLR) required by this module
+    # CLRVersion = ''
+
+    # Processor architecture (None, X86, Amd64) required by this module
+    # ProcessorArchitecture = ''
+
+    # Modules that must be imported into the global environment prior to importing this module
+    # RequiredModules = @()
+
+    # Assemblies that must be loaded prior to importing this module
+    # RequiredAssemblies = @()
+
+    # Script files (.ps1) that are run in the caller's environment prior to importing this module.
+    # ScriptsToProcess = @()
+
+    # Type files (.ps1xml) to be loaded when importing this module
+    # TypesToProcess = @()
+
+    # Format files (.ps1xml) to be loaded when importing this module
+    # FormatsToProcess = @()
+
+    # Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
+    NestedModules = @("Modules\MofHelper.psm1",
+        "Modules\PowerShellHelp.psm1",
+        "Modules\WikiPages.psm1")
```

Comment:
Oh I see it now. But I don't think it should be called 'Modules' in the root folder because there are other modules in the root that should not go in this folder as they are not part of this documentation helper. Can we make a folder for the documentation helper and put all these files inside a Modules folder under the DocumentaionHelper folder along with this manifest?